internal/backends/qt/qt_window.rs

```diff
@@ -1652,7 +1652,17 @@ impl PlatformWindow for QtWindow {
                 return QPointF();
             return QPointF(textLine.x() + textLine.cursorToX(offset), textLine.y());
         }};
-        Point::new(r.x as _, r.y as _)
+        Rect::new(
+            Point::new(r.x as _, r.y as _),
+            Size::new(
+                1.0,
+                text_input
+                    .unresolved_font_request()
+                    .merge(&self.default_font_properties())
+                    .pixel_size
+                    .unwrap(),
```

Comment:
Great, here you're using the method I think is indeed easiest. Please use that in the GL backend, too :)

Reply:
yeah. the thing is that it crashes. because i get None from the pixel_size and then the unwrap panics. at least in the gallery example.